packages/block-library/src/paragraph/edit.js

```diff
@@ -64,20 +65,54 @@ function ParagraphBlock( {
 		style: { direction },
 	} );
 
+	const updateContent = useCallback(
+		( newContent ) => setAttributes( { content: newContent } ),
+		[ setAttributes ]
+	);
+
+	const splitParagraph = useCallback(
+		( value, isOriginal ) => {
+			let newAttributes;
+
+			if ( isOriginal || value ) {
+				newAttributes = {
+					...attributes,
+					content: value,
+				};
+			}
+
+			const block = createBlock( name, newAttributes );
+
+			if ( isOriginal ) {
+				block.clientId = clientId;
+			}
+
+			return block;
+		},
+		[ attributes, createBlock ]
```

Comment:
We don't need `createBlock` here since it's a module export which will never change.

However, we need to add `clientId` here though.

Reply:
Good eye! I'm also going to mark this as resolved since I ended up rolling back changes